.buildkite/pipeline.yml

```diff
@@ -32,6 +32,16 @@ steps:
           namespace: "infra-cached-certificate-operator"
           env: "labs"
 
+  - label: ":wrench: Build Labs-Staging Image"
+    branches: "main"
+    plugins:
+      - ssh://git@github.com/pachama/docker-build-buildkite-plugin#v3.1.5:
```

Comment:
Not quite sure if `labs-staging` already works with the plugin (e.g. [here](https://github.com/pachama/docker-build-buildkite-plugin/blob/main/hooks/pre-command))
Could be though that since the agent now has access to GCR, this might work implicitly? 🤔